tapir/coop/tests/membership_resignation/test_service.py

```diff
@@ -87,8 +89,8 @@ def test_updateShiftsAndSharesAndPayOutDay_SharesAndPayOutDayForResignationTypeT
             ).count(),
             shares_after_update.count(),
         )
-        for share in shares_after_update.all():
-            self.assertEqual(share.end_date, resignation.cancellation_date)
+        # for share in shares_after_update.all():
```

Comment:
Therefore we don't need this test?!

Reply:
"This test" is the entire `test_updateShiftsAndSharesAndPayOutDay_SharesAndPayOutDayForResignationTypeTransfer_transferredAndSet` function, that we do need.

Instead of removing `self.assertEqual(share.end_date, resignation.cancellation_date)`, let's replace it with `self.assertEqual(share.end_date, None)`